set.mm

```diff
@@ -719932,6 +719985,169 @@ valid cases ( ` (/) ` is the last symbol) and invalid cases ( ` (/) `
+  Interchangeable setvar variables
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c <> $.
+
+  $( Extend wff notation to include the propery of a wff ` ph ` that the setvar
+     variables ` x ` and ` y ` are interchangeable.  Read this notation
+     as " ` x ` and ` y ` are interchangeable in wff ` ph ` ". $)
+  wich $a wff [ x <> y ] ph $.
+
+  ${
+    $d a ph $.  $d a x $.  $d a y $.
+    $( Define the propery of a wff ` ph ` that the setvar variables ` x ` and
+       ` y ` are interchangeable.  For an alternate definition using implicit
+       substitution and a temporary setvar variable see ~ ichcircshi .
+       Another, equivalent definition using two temporary setvar variables is
+       provided in ~ dfich2 .  (Contributed by AV, 29-Jul-2023.) $)
+    df-ich $a |- ( [ x <> y ] ph <-> A. x A. y
+                               ( [ x / a ] [ y / x ] [ a / y ] ph <-> ph ) ) $.
+  $}
+
+  ${
+    $d a ph $.  $d a x $.
+    $( A setvar variable is always interchangeable with itself.  (Contributed
+       by AV, 29-Jul-2023.) $)
+    ichid $p |- [ x <> x ] ph $=
+      ( va wich wsb wb wal sbid sbbii sbid2v bitri gen2 df-ich mpbir ) ABBDABCE
+      ZBBEZCBEZAFZBGBGRBBQOCBEAPOCBOBHIACBJKLABBCMN $.
+  $}
+
+  ${
+    $d z ph $.  $d x ps $.  $d y ch $.  $d x y z $.
+    ichcircshi.1 $e |- ( x = z -> ( ph <-> ps ) ) $.
+    ichcircshi.2 $e |- ( y = x -> ( ps <-> ch ) ) $.
+    ichcircshi.3 $e |- ( z = y -> ( ch <-> ph ) ) $.
+    $( The setvar variables are interchangeable if they can be circularily
+       shifted using a third setvar variable, using implicit substitution.
+       (Contributed by AV, 29-Jul-2023.) $)
+    ichcircshi $p |- [ x <> y ] ph $=
+      ( wich wsb wb wal weq bicomd equcoms sbievw 2sbbii sbbii 3bitri df-ich
+      gen2 mpbir ) ADEJAEFKZDEKFDKZALZEMDMUFDEUECDEKZFDKBFDKAUDCFDEDACEFACLFEFE
+      NCAIOPQRUGBFDCBDECBLEDEDNBCHOPQSBAFDBALDFDFNABGOPQTUBADEFUAUC $.
+  $}
+
+  ${
+    $d a b ph $.  $d z ph $.  $d a b x y $.  $d x y z $.
+    $( Definition of the propery of a wff ` ph ` that the setvar variables
+       ` x ` and ` y ` are interchangeable implies alternate definition.
+       (Contributed by AV, 6-Aug-2023.) $)
+    dfich2ai $p |- ( A. x A. y ( [ x / z ] [ y / x ] [ z / y ] ph <-> ph )
+                 -> ( [ a / x ] [ b / y ] ph <-> [ b / x ] [ a / y ] ph ) ) $=
+      ( wsb wb wal sbco4 bicomi bibi1i 2albii nfs1v nfv sbbib albii alcom sylbi
+      nfsb bitri 2sp ) ACDGBCGDBGZAHZCIBIACFGZBEGZFBGZECGZAHZCIZBIZUFACEGZBFGHZ
+      UDUIBCUCUHAUHUCABCDFEJKLMUKUGULHZEIZBIZUMUJUOBUGAECUFFBCUEBECACFNTTAEOPQU
+      PUMFIZEIZUMUPUNBIZEIURUNBERUSUQEUFULFBUEBENULFOPQUAUMEFUBSSS $.
+
+    $d a b z $.
+    $( Alternate definition of the propery of a wff ` ph ` that the setvar
+       variables ` x ` and ` y ` are interchangeable implies definition (see
+       also ~ wl-ich ).  (Contributed by AV, 6-Aug-2023.) $)
```

Comment:
wl-ich is going to be deleted sooner or later.  It served as an ad hoc solution to your public request, but is of no interest on its own, I think.

Reply:
You can remove the reference from the comment as soon as you delete the theorem. I would like, however to use your proof, because its shorter. Shall I take/move it from your mathbox (together with the required lemmata?) Of course you will still be the contributor.

Reply:
ok